### PR TITLE
Add testnet-4 config

### PR DESCRIPTION
**Pull request by rickmak**

### CODE REVIEW

File: likecoin-public-testnet-4/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+chain-id: `likecoin-public-testnet-4`
+
+codename: FoTan
+
+## Current Software Version
+
+[v1.2.0](https://github.com/likecoin/likecoin-chain/releases/tag/v1.2.0) [`aabee90550df7884ead7e5f8eece69e8cec964ca`](https://github.com/likecoin/likecoin-chain/commit/aabee90550df7884ead7e5f8eece69e8cec964ca)
+
+Base on Cosmos SDK v0.42.7
```
Comment (williamchong) [inline likecoin-public-testnet-4/README.md]:
```suggestion
Base on Cosmos SDK v0.42.11
```

File: likecoin-public-testnet-4/network.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "id": "likecoin-public-testnet-4",
+  "name": "LikeCoin public test chain",
+  "description": "LikeCoin is a decentralized publishing infrastructure. It provides a comprehensive metadata framework to facilitate content registration, licensing, and monetization for all media types.",
+  "logo": "logo.svg",
+  "website": "https://rinkeby.like.co",
+  "siteURL": "https://stake.like.co",
+  "apiURL": "https://likecoin-public-testnet-4.oursky.dev/",
+  "rpcURL": "https://likecoin-public-testnet-4.oursky.dev/rpc/",
+  "authcoreURL": "https://authcore.like.co",
+  "stakingWalletURL": "https://stake.like.co",
```
Comment (williamchong) [inline likecoin-public-testnet-4/network.json]:
```suggestion
  "stakingWalletURL": "https://likecoin-public-testnet-4.netlify.app",
```

File: likecoin-public-testnet-4/network.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "id": "likecoin-public-testnet-4",
+  "name": "LikeCoin public test chain",
+  "description": "LikeCoin is a decentralized publishing infrastructure. It provides a comprehensive metadata framework to facilitate content registration, licensing, and monetization for all media types.",
+  "logo": "logo.svg",
+  "website": "https://rinkeby.like.co",
+  "siteURL": "https://stake.like.co",
```
Comment (williamchong) [inline likecoin-public-testnet-4/network.json]:
```suggestion
  "siteURL": "https://likecoin-public-testnet-4.netlify.app",
```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: likecoin-public-testnet-4/keplr.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  "chainName": "LikeCoin public test chain",
+  "rpc": "https://likecoin-public-testnet-4.oursky.dev/rpc/",
+  "rest": "https://likecoin-public-testnet-4.oursky.dev/",
+  "stakeCurrency": {
+    "coinDenom": "EKIL",
+    "coinMinimalDenom": "nanoekil",
+    "coinDecimals": 9,
+    "coinGeckoId": "likecoin"
+  },
+  "walletUrlForStaking": "https://stake.like.co",
```
Comment (williamchong) [inline likecoin-public-testnet-4/keplr.json]:
```suggestion
  "walletUrlForStaking": "https://likecoin-public-testnet-4.netlify.app",
```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: likecoin-public-testnet-4/README.md (hunk 1 above)
Comment (rickmak) [reply likecoin-public-testnet-4/README.md]: Oh, yes. sorry.

**Comment by williamchong** [comment]
thanks

**Pull request merged**